src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/navigate-modal/NavigateModal.scss

```diff
@@ -0,0 +1,80 @@
+@import 'src/styles/common';
+
+.browserNavBarControls {
+  margin: 15px 0 0 40px;
+
+  .navButtonWrapper {
+    display: inline-block;
+    margin-right: 35px;
+  }
+}
+
+.inputGroup {
+  margin-top: 10px;
+  height: 30px;
+  font-weight: $light;
+
+  label {
+    & > span {
+      display: inline-block;
+      margin-right: 10px;
+      text-align: right;
+      width: 32px;
+    }
+  }
+
+  input {
+    border: 1px solid $light-grey;
+    height: 30px;
+    padding: 2px 6px;
+  }
+}
+
+.coordInputs {
+  margin-top: 20px;
+
+  input {
+    width: 140px;
+  }
+}
+
+.regionInput {
+  margin-top: 30px;
+
+  input {
+    width: 230px;
+  }
+}
+
+.moveLeftButton,
+.moveRightButton {
+  height: 22px;
+  width: 22px;
+}
+
+.zoomInButton,
+.zoomOutButton,
+.browserReset {
+  height: 16px;
+  width: 16px;
+}
```

Comment:
By doing so, aren't you now competing with the selectors of the same specificity on these buttons? If so, then the styles will be relying on the order in which CSS rules are declared, which will eventually rely on the order in which Webpack puts together the CSS files.

Is this perhaps an argument for the use of CSS variables?

Reply:
✅